src/components/PostPageDetail/index.js

```diff
@@ -105,6 +103,7 @@ const PostPageDetailIdComponent = (props) => {
   React.useEffect(() => {
     getComment();
   }, []);
+
   const handleVote = (data = {}) => {
     const upvote = data.upvotes ? data.upvotes : 0;
     const downvotes = data.downvotes ? data.downvotes : 0;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `handleVote` function now handles default values for `upvotes` and `downvotes`. This is a good practice as it prevents potential issues if `upvotes` or `downvotes` are undefined.
```diff
  const handleVote = (data = {}) => {
-    const upvote = data.upvotes ? data.upvotes : 0;
-    const downvotes = data.downvotes ? data.downvotes : 0;
+    const upvote = data.upvotes || 0;
+    const downvotes = data.downvotes || 0;
  }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->